### ISSUE TITLE
Confirm label coding

### ISSUE THREAD

**Issue by Katiedaisey** [delegability/data#1]
Great ideas and a wonderful survey.

Could you please confirm the encoding of `label`  in the csv files and used in the analyses are as:

1. Human Only
2. Machine-in-the-loop
3. Human-in-the-loop
4. Machine Only

This seems opposite of the encoding in the survey and in the `Plots` page of the overview.

**Comment by Katiedaisey** [comment]
@blubars Any update?

**Comment by blubars** [comment]
@Katiedaisey I'm so sorry, missed this before! Thank you for bringing this up. 

The label in the CSV here is the same as the encoding from the survey (the only processing we did was to remove identifying info and attention check questions), but you're right that we reversed it for our analysis in the paper and website. We flipped it in the analysis so that it aligned better with the word delegability, where a more delegable task means less human control. Note the survey encoding was not visible to the survey takers, where the options were presented in an unordered list.

So:
* [Surveys](https://github.com/delegability/surveys) and [CSVs](https://github.com/delegability/data) encode 4 as Human Only, 1 as Machine Only. 
* Analysis in paper and website uses 1 as Human Only, 4 as Machine Only.

Apologies for the confusion! I will add a readme to this repo clarifying the encodings.
